### ISSUE TITLE
Kirby's Return to Dream Land level map backdrops brightness issue

### ISSUE THREAD

**Issue by CheezyFriez12** [magcius/noclip.website#500]
The level map backdrops aren't as bright as they are in game.

**Comment by magcius** [comment]
Yeah, I'm not emulating any postprocessing / lighting that Kirby is doing right now. If anybody wants to help make it more accurate, that would be lovely!
